source/mouseHandler.py

```diff
@@ -379,3 +379,32 @@ def unlockRightMouseButton():
 	# Translators: This is presented when the right mouse button lock is released (used for drag and drop).
 	ui.message(_("Right mouse button unlock"))

```

Comment:
I know this is old, but...

```suggestion
	ui.message(_("Right mouse button unlocked"))
```

I would like to do the left one as well, but when expanding up to it, GH won't allow a change suggestion.